dotnet-desktop-guide/framework/wpf/advanced/how-to-recognize-application-gestures.md

```diff
@@ -8,6 +8,7 @@ helpviewer_keywords:
   - "application gestures [WPF], recognizing"
   - "gestures [WPF], recognizing"
 ms.assetid: d58b740f-5192-4a3e-af59-7aa162e6ca15
+description: Learn how to erase ink when a user makes a ScratchOut gesture on an InkCanvas, assuming an InkCanvas, called inkCanvas1, is declared in the XAML file.
```

Comment:
If I am reading this correctly, the comma after the 2nd mention of "InkCanvas" should be removed.